recipe/bld.bat

```diff
@@ -0,0 +1,6 @@
+
+nmake /f makefile.vc
+if errorlevel 1 exit 1
+
```

Comment:
Thanks for the explanation. (I thought I was starting to understand how to build stuff on Windows :smiley_cat:)